CricketNotifications.py

```diff
@@ -13,21 +15,26 @@
 series_names=[]
 datapath=[]
 for i in temp:
+    
```

Comment:
not here actually :rofl: 
I meant above `for loop` for loop is treated as a single block of code.
we are to break code into code blocks